src/main/haskell/kore/src/Kore/Builtin/KEqual.hs

```diff
@@ -113,3 +154,44 @@ evalKEq true false tools substitutionSimplifier _ pat =
       where
         ep1 = ExpandedPattern.fromPurePattern t1
         ep2 = ExpandedPattern.fromPurePattern t2
+
+evalKIte
+    ::  forall variable
+    .   ( FreshVariable variable
+        , Hashable variable
+        , OrdMetaOrObject variable
+        , SortedVariable variable
+        , ShowMetaOrObject variable
+        )
+    => MetadataTools.MetadataTools Object StepperAttributes
+    -> PredicateSubstitutionSimplifier Object Simplifier
+    -> PureMLPatternSimplifier Object variable
+    -> Application Object (PureMLPattern Object variable)
+    -> Simplifier
+        ( AttemptedFunction Object variable
+        , SimplificationProof Object
+        )
+evalKIte _ _ _ pat =
+    case pat of
+        Application
+            {  applicationChildren = [expr, t1, t2] } ->
+            evalIte expr t1 t2
+        _ -> notApplicableFunctionEvaluator
+  where
+    evaluate :: Functor.Foldable.Fix (Pattern Object variable) -> Maybe Bool
+    evaluate expr = case Functor.Foldable.project <$> [expr] of
+        [DomainValuePattern a] -> Just (get a)
+        _                      -> Nothing
+
+    get :: DomainValue Object (BuiltinDomain child) -> Bool
+    get =
+        Builtin.runParser (Text.unpack "ite")
```

Comment:
The call to `Text.unpack` should not be necessary. Please use the full hook name of the builtin; some builtin domains have overlapping function names, and I think this is one.
```suggestion
        Builtin.runParser "KEQUAL.ite"
```